### PR TITLE
Add travis badge

### PR DESCRIPTION
**Pull request by tareqmamari**

### CODE REVIEW

**Comment by csantanapr** [comment]
@tareqmamari It looks like Travis is working, and it marked your commit with an x, please check what's the error

**Comment by tareqmamari** [comment]
Now it is fixed :)

File: tests/credentials.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "credentials": {
             "apiKey": "XXXXXXXX",
             "apiToken": "YYYYYYY",
-            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1",
+            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1"
```
Comment (csantanapr) [inline tests/credentials.json]: please use a more generic uri (i.e. no ibm stuff) 
maybe change to "base_uri": "https://api.service-x.com/v1"

File: tests/credentials.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
         "credentials": {
             "apiKey": "ZZZZZZZZZZ",
             "apiToken": "MMMMMMMMMM",
-            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1",
+            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1"
```
Comment (csantanapr) [inline tests/credentials.json]: please use a more generic uri (i.e. no ibm stuff) 
maybe change to "base_uri": "https://api.service-y.com/v1"

**Comment by csantanapr** [comment]
@tareqmamari address the review comments and I will merge.

**Comment by csantanapr** [comment]
rebase and merged 57ba4fb922ef65c2946132ae3111554b2f029118

**Pull request closed without merging**